### PR TITLE
Map module

### PR DESCRIPTION
**Pull request by Gusvarov**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -3,7 +3,6 @@ import { SignUpComponent } from './component/user/auth/sign-up/sign-up.component
 import { AuthComponent } from './component/user/auth/auth.component';
```
Comment (nabermi2020) [inline src/app/app-routing.module.ts:3]: general: please move map directory to the root level(the same level as eco-news directory)

Comment (Gusvarov) [reply src/app/app-routing.module.ts:3]: fixed

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -3,7 +3,6 @@ import { SignUpComponent } from './component/user/auth/sign-up/sign-up.component
...
 import { SignInComponent } from './component/user/auth/sign-in/sign-in.component';
 import { SubmitEmailComponent } from './component/user/auth/submit-email/submit-email.component';
-import { MapComponent } from './component/user/map/map.component';
```
Comment (nabermi2020) [inline src/app/app-routing.module.ts:6]: Please add lazy loading to the map module

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (Gusvarov) [reply src/app/app-routing.module.ts:3]: fixed

File: src/app/component/map/map.module.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import { NgModule } from '@angular/core';
+import { MapComponent } from './map-component/map.component';
+import {
+    DeleteFavoriteComponent,
+    EditFavoriteNameComponent,
+    FavoritePlaceComponent
+  } from '../user/favorite-place/favorite-place.component';
```
Comment (RomanDeveloperAcc) [inline src/app/component/map/map.module.ts:7]: Please delete space at the beginning

Comment (Gusvarov) [reply src/app/component/map/map.module.ts:7]: fixed

File: src/app/app-routing.module.ts [hunk 4]
```diff
@@ -3,7 +3,6 @@ import { SignUpComponent } from './component/user/auth/sign-up/sign-up.component
 import { AuthComponent } from './component/user/auth/auth.component';
 import { SignInComponent } from './component/user/auth/sign-in/sign-in.component';
 import { SubmitEmailComponent } from './component/user/auth/submit-email/submit-email.component';
```
Comment (nabermi2020) [inline src/app/app-routing.module.ts:5]: general: please move all components which are used in the map module to the map directory

File: src/app/component/filter/filter.component.ts [hunk 5]
```diff
@@ -2,7 +2,7 @@ import {Component} from '@angular/core';
 import {Options} from 'ng5-slider';
 import {PlaceService} from '../../service/place/place.service';
 import {FilterPlaceService} from '../../service/filtering/filter-place.service';
-import {MapComponent} from '../user/map/map.component';
+import {MapComponent} from '../map/map-component/map.component';
```
Comment (KimAT25) [inline src/app/component/filter/filter.component.ts:5]: add spaces between brackets

File: src/app/app-routing.module.ts (hunk 4 above)
Comment (Gusvarov) [reply src/app/app-routing.module.ts:5]: fixed

File: src/app/component/filter/filter.component.ts (hunk 5 above)
Comment (Gusvarov) [reply src/app/component/filter/filter.component.ts]: fixed

File: src/app/component/auth/auth.module.ts [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import { NgModule } from '@angular/core';
 import { NewSignUpComponent } from './new-sign-up/new-sign-up.component';
-import {SignInNewComponent} from './sign-in-new/sign-in-new.component';
-import {CoreModule} from '../core/core.module';
+import { SignInNewComponent } from './sign-in-new/sign-in-new.component';
```
Comment (nabermi2020) [inline src/app/component/auth/auth.module.ts:3]: Minor: Please don't  make any changes to files that aren't related to your direct ticket

**Pull request merged**